src/modules/85-cv/pages/monitored-service/components/Configurations/Configurations.tsx

```diff
@@ -44,7 +44,10 @@ interface ConfigurationsInterface {
   updateTemplate?: (template: NGTemplateInfoConfigWithMonitoredService) => Promise<void>
 }
 
-export default function Configurations({ isTemplate, updateTemplate }: ConfigurationsInterface): JSX.Element {
+export default function Configurations(
+  { isTemplate, updateTemplate }: ConfigurationsInterface,
+  formikRef: any
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9306981)
```suggestion
  formikRef: unknown
```